### PR TITLE
Update examples to 1.13.x standards

### PR DESCRIPTION
**Pull request by typeoneerror**
* No Ember.View
* No Ember.keys

### CODE REVIEW

File: source/templates/binding-element-attributes.md [hunk 1]
```diff
@@ -79,23 +79,19 @@ Now the same handlebars code above renders the following HTML:
...
-    this._super();
-    var self = this;
-
-    // bind attributes beginning with 'data-'
-    Ember.keys(this).forEach(function(key) {
-      if (key.substr(0, 5) === 'data-') {
-        self.get('attributeBindings').pushObject(key);
+export default Ember.Component.reopen({
+  bindAttributes: Ember.on('init', function() {
+    Object.keys(this).forEach((key) => {
```
Comment (typeoneerror) [inline source/templates/binding-element-attributes.md]: As mentioned in the previous pull request, the use here of Object.keys over Ember.keys is moot. Deprecation warnings are thrown if Ember.keys is used, so this seems correct, but is it?

File: source/templates/binding-element-attributes.md [hunk 2]
```diff
@@ -79,23 +79,19 @@ Now the same handlebars code above renders the following HTML:
...
-export default Ember.View.reopen({
-  init() {
-    this._super();
-    var self = this;
-
-    // bind attributes beginning with 'data-'
-    Ember.keys(this).forEach(function(key) {
-      if (key.substr(0, 5) === 'data-') {
-        self.get('attributeBindings').pushObject(key);
+export default Ember.Component.reopen({
```
Comment (locks) [inline source/templates/binding-element-attributes.md]: someone raised a question about exporting a `.reopen`, I think this snippet needs to be reviewed.

cc @mixonic @michaelrkn

Comment (typeoneerror) [inline source/templates/binding-element-attributes.md]: yeah, considered removing that as well. does seem odd.

Comment (michaelrkn) [inline source/templates/binding-element-attributes.md]: @mmun says to just remove that whole section :)

Comment (typeoneerror) [inline source/templates/binding-element-attributes.md]: @michaelrkn ok, one sec.

Comment (typeoneerror) [inline source/templates/binding-element-attributes.md]: @mmun @michaelrkn removed data- example and the exports w/reopen

**Comment by michaelrkn** [comment]
Awesome!

**Pull request merged**